src/service/tag-service.js

```diff
@@ -0,0 +1,22 @@
+import TagRepogitory from '../repository/tag-repository.js';
+
+class TagService {
+  async getTags(page, limit, order, search) {
+    const where = {
+      name: { contains: search, mode: 'insensitive' },
+    };
+
+    const skip = (page - 1) * limit;
+    const take = limit;
+
+    return await TagRepogitory.getTags(where, order, skip, take);
```

Comment:
```suggestion
    const tags = TagRepogitory.getTags(where, order, skip, take);

    return {tags, totalCount};
```
태그 정보와 총 갯수를 따로 받아서 같이 반환합니다.